struts2-jquery-plugin/src/main/resources/template/jquery/autocompleter.ftl

```diff
@@ -18,27 +18,40 @@
  * under the License.
  */
 -->
+<#assign hasFieldErrors = parameters.widgetname?? && fieldErrors?? && fieldErrors[parameters.widgetname]??/>
+<input type="hidden"
+    <#if parameters.widgetid?if_exists != "">
+        id="${parameters.widgetid?html}"<#rt/>
+    </#if>
+    <#if parameters.nameValue??>
+        value="<@s.property value="parameters.nameValue"/>"<#rt/>
+    </#if>
+    <#if parameters.widgetname?if_exists != "">
+        name="${parameters.widgetname?html}"<#rt/>
+    </#if>
+    <#if parameters.cssClass?has_content && !(hasFieldErrors && parameters.cssErrorClass??)>
+        class="${parameters.cssClass?html}"<#rt/>
+    <#elseif parameters.cssClass?has_content && (hasFieldErrors && parameters.cssErrorClass??)>
+        class="${parameters.cssClass?html} ${parameters.cssErrorClass?html}"<#rt/>
+    <#elseif !(parameters.cssClass?has_content) && (hasFieldErrors && parameters.cssErrorClass??)>
+        class="${parameters.cssErrorClass?html}"<#rt/>
+    </#if>
+    <#if parameters.cssStyle?has_content && !(hasFieldErrors && (parameters.cssErrorStyle?? || parameters.cssErrorClass??))>
+        style="${parameters.cssStyle?html}"<#rt/>
+    <#elseif hasFieldErrors && parameters.cssErrorStyle??>
+        style="${parameters.cssErrorStyle?html}"<#rt/>
+    </#if>
+    <#if parameters.disabled?default(false)>
+        disabled="disabled"<#rt/>
+    </#if>
+/>

```

Comment:
won't this create a problem when the parentTheme `xhtml` or `css_xhtml` is used?

the controlheaders will be added below the input field
```ftl
        <#if parameters.parentTheme == 'xhtml'>
  		<#include "/${parameters.templateDir}/xhtml/controlheader.ftl" />
  	</#if>
  	<#if parameters.parentTheme == 'css_xhtml'>
  		<#include "/${parameters.templateDir}/css_xhtml/controlheader.ftl" />
  	</#if>
```